tests/hawc/apps/assessment/test_view_permissions.py

```diff
@@ -15,8 +15,7 @@
     "noel_name": 0,
     "rob_name": 0,
     "editable": "on",
-    "project_manager_0": ("2",),
-    "project_manager_1": ("2",),
+    "project_manager": ("2",),
```

Comment:
i guess we can revert this change too, will occur when we actually roll this out